graphql/tests/taskQueueDistros/data.json

```diff
@@ -0,0 +1,72 @@
+{
```

Comment:
Could you run the "make scramble" script on the data you added here? If there are specific fields you want to keep as is you can add them to the list in https://github.com/evergreen-ci/evergreen/blob/master/cmd/scrambled-eggs/scramble.py#L65

Reply:
![Screen Shot 2020-08-26 at 9 36 14 AM](https://user-images.githubusercontent.com/15262143/91310247-9da63600-e77f-11ea-858a-c5f1e607ab26.png)

I get this error. There is also no sensitive information in this file.

Reply:
The point is to run it so that you never have to worry about if there's sensitive information. The syntax to run it is `make scramble file=graphql/tests/taskQueueDistros/data.json`

Reply:
i changed the project names which are the only sensitive fields

Reply:
What we talked about was adding keys to the list of fields that aren't scrambled, then running it. Could you do that?

Reply:
awesome!!! done